hack/go-build.sh

```diff
@@ -9,3 +9,4 @@ set -x
 
 go build -a -o ${GOBIN:-bin}/manager main.go
 go build -a -o ${GOBIN:-bin}/status-reporter ./service/status-report/main.go
+go build -a -o ${GOBIN:-bin}/csi-image-set-creator ./service/csi-image-set-creator/main.go
```

Comment:
lets update the build team for this new change

Reply:
Sorry, I am still confused. Why are we pushing code that needs to happen in build time into runtime? 
I asked on multiple reviews to have the config maps be produced at build time, there is no point in having a container do a build time operation at runtime.

Reply:
as commented in https://github.com/red-hat-storage/ocs-client-operator/pull/175#discussion_r1692578565

If we are not going to use csi-op then we need to revert the d/s build time code. Nevertheless, I'll ask build time to have both types of ConfigMaps be bundled.